examples/kafka-plugin/hash-partitioner/main.go

```diff
@@ -186,7 +193,7 @@ func (plugin *ExamplePlugin) closeExample() {
 
 			plugin.Log.Info("kafka example finished, sending shutdown ...")
 
-			plugin.closeChannel <- struct{}{}
+			plugin.exampleFinished <- struct{}{}
```

Comment:
Did this example even stop automatically? Because the `exampleFinished` should be closed and not send data to it.

Reply:
Never mind.. realized the channel doesn't differ between send and close... which to me seems more like unintended behavior.